using-mobile-extensions/adobe-analytics-edge/product-variable.md

```diff
@@ -57,9 +57,9 @@ NSMutableDictionary *contextData = [NSMutableDictionary dictionary];
 
```

Comment:
The  Implementing a Merchandising Variable link no longer works, I think this is the new link https://experienceleague.adobe.com/docs/analytics/implementation/vars/plugins/addproductevar.html